roles/system/tasks/main.yml

```diff
@@ -3,22 +3,23 @@
 - name: Apply k8s resources
   include_tasks: k8s-upsert.yaml
   with_fileglob:
-    - "templates/configmaps/*.yaml"
-    - "templates/deployments/*.yaml"
-    - "templates/ingresses/*.yaml"
-    - "templates/podmonitors/*.yaml"
-    - "templates/secretdefinitions/*.yaml"
-    - "templates/services/*.yaml"
-    - "templates/statefulsets/*.yaml"
+    - "templates/system/configmaps/*.yaml"
```

Comment:
Why have been renamed the directory of the templates? all this templates are already on `roles/system/templates`, why it is needed a second `system` preffix so having `roles/system/templates/system`?

I understand that now `sphinx` is optional (like pdb or hpa), but it is still part from system.

Actually, with current naming (without that new role `system` preffix), tasks file can be almost he same on all roles, by only having a few differences on optional objects.

Reply:
To make a clearer the separation between both workloads, can be done in a single folder and selecting each yaml instead of using wildcards. I think is clearer this way, but don't mind to change it back.

In case that the sphinx is opted out, the stateful set and the service will not be deployed. Those yamls live in the `/sphinx` folder. The sphinx config map lives in the  `/system` folder instead as is required for the system workload even when sphinx is opted out, as it contains the external sphinx endpoint.

What do you thing @roivaz ?

Reply:
After some internal discussion, the sphinx and system files will be mixed into the same resource-type based directories as the other system workloads. And the fine grain opt-in / opt-out logic will be implement at task level.

https://github.com/3scale/saas-operator/pull/48/commits/6476f795f112f22f6aa4abeaf473ab9634f4e2c0